### ISSUE TITLE
`bun x` seems to ignore `postinstall`

### ISSUE THREAD

**Issue by MonsterDeveloper** [oven-sh/bun#8154]
### What version of Bun is running?

1.0.22+b400b36ca

### What platform is your computer?

Darwin 23.0.0 arm64 arm

### What steps can reproduce the bug?

In the terminal, run `bunx inngest-cli@latest dev`

### What is the expected behavior?

Inngest dev server starting (as with npx: `npx inngest-cli@latest dev`)

### What do you see instead?

`error: Failed to run "inngest-cli" due to error InvalidExe`

### Additional information

`inngest-cli` [uses a postinstall script](https://github.com/inngest/inngest/blob/main/npm/postinstall.ts) to download the correct binary. Bun appears to ignore it when launching the CLI with `bunx`.

After adding `inngest-cli` to `devDependencies` and `trustedDependencies`, the postinstall script is executed normally, and starting the CLI with Bun works.

**Comment by RiskyMH** [comment]
Yeah this is Bun doesn't run postinstall scripts by default, unless listed in your `trustedDependencies` or [default list](https://github.com/oven-sh/bun/blob/main/src/install/default-trusted-dependencies.txt). For this case as the package seems popular enough we should add it to the list.

_see https://bun.sh/docs/install/lifecycle#trusteddependencies_
